packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -384,7 +398,7 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         // TODO Show header, when we can have entries in-line with timeline-chart
         return <>
             <div ref={this.timeGraphTreeRef} className='scrollable' onScroll={() => this.synchronizeTreeScroll()}
-                style={{ height: parseInt(this.props.style.height.toString()) - this.getMarkersLayerHeight() }}
+                style={{ height: parseInt(this.props.style.height.toString()) - this.getMarkersLayerHeight() - (document.getElementById(this.props.traceId + this.props.outputDescriptor.id + 'searchBar')?.offsetHeight ?? 0)}}
```

Comment:
I get a linter warning here:
This line has a length of 224. Maximum allowed is 180.eslint[max-len](https://eslint.org/docs/rules/max-len)